src/LookingGlassWindow.ts

```diff
@@ -0,0 +1,50 @@
+import { LookingGlassConfig } from './LookingGlassConfig';
```

Comment:
This file moves the window creation logic out of the `LookingGlassWebGLLayer.ts` file, this allows that file to solely handle WebGL issues and cleans up the code base.